test_data_e2e/generate_sha1sum_files.py

```diff
@@ -0,0 +1,20 @@
+import os
+
+from scripts.checksum import compute_checksum
+
+
+def compute_checksum_in_folder(top_dir, algorithm: str):
```

Comment:
Is there a need for algorithm to be a parameter here? It should be always 'sha1', right?

Reply:
indeed it doesn't need to be configurable here. I will change it since we have been using sha1 sums only.